### PR TITLE
Oracle NoSQL: Resolve the issues raised by the code quality linters.

### PR DESCRIPTION
**Pull request by dario-vega**
I have executed the script `lint` provided in PR 2513 (improve the code quality linters)
- I resolved the issues detected in `NoSQL` directory
- I resolved the issues detected in `.github/workflows/build-and-push-nosql-*`. FI: There are other files in this directory with warnings.

I have also resolved the following warnings in GitHub Workflows
- Node.js 12 actions are deprecated (`actions/checkout@v2`)
- `set-output` command is deprecated and will be disabled soon

I have fixed an issue in `setup-http-proxy-sec.sh`
- I was generating the `driver-trust` with a wrong password

Signed-off-by: Dario Vega [dario.vega@oracle.com](mailto:dario.vega@oracle.com)

### CODE REVIEW

**Review by Djelibeybi — APPROVED** [review]
This is awesome. I'm very pleased to be able to approve a PR without a single review comment! 😂

**Comment by Djelibeybi** [comment]
I'll need at least @mikebrey to also approve (as the other code owner for the NoSQL content) before I   merge.

**Review by mikebrey — APPROVED** [review]
changes look good.

**Pull request merged**
